rust/src/delta.rs

```diff
@@ -1034,6 +1059,31 @@ impl DeltaTable {
         })
     }
 
+    /// Create a DeltaTable with version 0 given the provided MetaData and Protocol
+    pub async fn create(
+        &mut self,
+        metadata: DeltaTableMetaData,
+        protocol: action::Protocol,
+    ) -> Result<(), DeltaTransactionError> {
+        let meta = action::MetaData::try_from(metadata)?;
+
+        // TODO add commit info action
```

Comment:
do you mind filing a follow up issue for this?

Reply:
I will, do you know where I can find some more info related to commit info? Is that something that still hasn't been fully implemented in delta-rs?

Reply:
everything you want to know about the delta spec is documented at https://github.com/delta-io/delta/blob/master/PROTOCOL.md#commit-provenance-information. It's modeled as a json object in the current code base: https://github.com/delta-io/delta-rs/blob/main/rust/src/action.rs#L720